FCCeeAnalyses/ZH_Zee/dataframe/preSel.py

```diff
@@ -1,6 +1,9 @@
+from common_defaults import deffccdicts
+
 #python FCCeeAnalyses/ZH_Zmumu/dataframe/preSel.py 
+import os
 
-basedir="/afs/cern.ch/work/h/helsens/public/FCCDicts/yaml/FCCee/fcc_v01/"
+basedir=os.path.join(os.getenv('FCCDICTSDIR', deffccdicts), '') + "yaml/FCCee/fcc_v01/"
```

Comment:
```suggestion
basedir=os.path.join(os.getenv('FCCDICTSDIR', deffccdicts), 'yaml/FCCee/fcc_v01/')
```

should be the same thing, but slightly shorter? The original version is fine by me too, though.

Reply:
Uhm ... the first form ensures that there is a ending '/' ... not sure about the second ... but we can have
 '/yaml/FCCee/fcc_v01/' ...